### PR TITLE
What's New Page for version 64 (Fixes #6492)

### LINKED ISSUE #6492: The WNP 64 page
| Estimates |  |
| ------------- | ------------- | 
| TL | |
| PM | |
| ENG | |
| SEO | |
| ANA | |
| DES | |
| COPY | |

Description: create the WNP promoting Firefox Account features/benefits reusing as much content from the FxA Benefits page as possible.

Pull request - https://github.com/mozilla/bedrock/pull/6513

---
#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Strings must be final and in a pull request by EOD 11/21
- [ ] Must go live in production on or prior to 11/30

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] Timing  . . .

Tasks
- [ ] UX/UI/Design (@mrkwvr)
- [ ] Submit PR w. strings on 11/21
- [ ] QA testing - Design (Internal team)
- [ ] QA testing - Design - Stakeholder (Michele W, Lauren N.)
- [ ] Code page w. design
- [ ] QA testing - coded page (Internal team)
- [ ] QA testing - coded page - Stakeholder (Michele W, Lauren N.)

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description
Add What's New Page for version 64

## Issue / Bugzilla link
Fixes #6492

## Testing
- buttons open modals with QR codes that go to the right place

### CODE REVIEW

**Comment by craigcook** [comment]
image

The vertical spacing between one block and the next is pretty tight on mobile, I think they could use a bit of bottom-margin (not too much, scrolling on mobile sucks).

**Comment by ejregithub** [comment]
Do we care so much about small screens on a page shown just with the desktop product?

**Comment by craigcook** [comment]
> Do we care so much about small screens on a page shown just with the desktop product?

It's not a showstopper, no, but it's also not hard to change and is just something I noticed.

**Comment by stephaniehobson** [comment]
Note to self: add `aria-role="button"` to the links when the JS inits.

File: media/css/firefox/whatsnew/whatsnew-64.scss [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+        .c-page-header-logo-moz {
+            @include bidi(((float, right, left),));
+            display: block;
+            margin: $spacing-md 0;
+        }
+
+        &.is-up-to-date .c-page-header-inner {
+            align-items: center;
+            display: flex;
+            justify-content: space-between;
```
Comment (craigcook) [inline media/css/firefox/whatsnew/whatsnew-64.scss]: ❤️

**Comment by craigcook** [comment]
image

Another spot that could use a little space, between the QR code and store button.

**Comment by stephaniehobson** [comment]
Updated.

**Pull request merged**